README.md

```diff
@@ -58,7 +114,7 @@ the download key (and possibly the collection key) into the asynchronous functio
 status can be updated.
 
 Arguments:
-- `pk`: the unique identifier for a collection of downloads - this will typically be a user PK
+- `username`: the unique identifier for a collection of downloads - this will typically be a username
```

Comment:
Changed that, because I added exception that user cannot be `string` from Dave suggestions. 
We actually need that possibility because across suite we can have different user pks.

And responding to your question: user can be still pk for back compatibility.